kano_settings/set_keyboard.py

```diff
@@ -73,19 +73,19 @@ class SetKeyboard(Template):
     selected_country_index = 21
     selected_variant = None
     selected_variant_index = 0
-    selected_continent_hr = "America"
-    selected_country_hr = "USA"
-    selected_variant_hr = "generic"
+    selected_continent_hr = 'America'
+    selected_country_hr = 'USA'
+    selected_variant_hr = 'generic'
```

Comment:
We agreed not to translate any keyboard settings as there are a LOT of countries and it was considered that users on the whole would not be swapping their keyboards away from the Kano supplied one.